src/main/java/interface_adapters/ConsoleEventPresenter.java

```diff
@@ -0,0 +1,31 @@
+package main.java.interface_adapters;
+
+import main.java.use_case.CalendarEventPresenter;
+import main.java.use_case.EventInfo;
+
+import java.util.List;
+
+public class ConsoleEventPresenter implements CalendarEventPresenter {
+
+    @Override
+    public void presentEvents(List<EventInfo> eventInfos) {
+        if (eventInfos.size() == 0) {
+            System.out.println("No events have been created");
+        }
+        for (EventInfo ei : eventInfos) {
+
+            String name = ei.getName();
+            String startTime = ei.getStartTime().toString();
+            String endTime = ei.getEndTime().toString();
+            String tags = ei.getTags().toString();
+            String dates = ei.getDates().toString();
+
+            String output = "Event: " + name + ", "
+                    + "start time = " + startTime + ", "
+                    + "end time = " + endTime + ", "
+                    + "tags = " + tags + ", "
+                    + "dates = " + dates;
+            System.out.println(output);
```

Comment:
We could have a toString() method inside EventInfo and use it here

Reply:
Personally I want to deny this suggestion. The concrete presenters are responsible for formatting data taken from the DTO. Formatting is not the DTO's responsibility, especially since it'll be coupled to specifically console output (when we plan on adding GUI output). I think there's some room for debate, though. Just follow up with some reasons why you think we should have a `toString()` and we can discuss it